checklist.md

```diff
@@ -35,17 +36,20 @@ To view the corresponding web guide section for each area of the checklist,
 * Don't use `!important`
 * Don't use `@import` in plain CSS
 * Avoid browser-specific hacks
+* Use relative over absolute units
+* Don’t generate content with CSS
```

Comment:
This curly quote might cause an encoding problem, convert to singlequote to match others.